server/src/main/java/org/apache/iotdb/db/metadata/schemaregion/SchemaRegion.java

```diff
@@ -165,6 +166,7 @@
   // device -> DeviceMNode
   private LoadingCache<PartialPath, IMNode> mNodeCache;
   private TagManager tagManager;
+  private MetadataSyncManager syncManager = MetadataSyncManager.getInstance();
```

Comment:
in the commit "add TsFileSyncManager", i re-implement the data collect process in storage engine.
1. just like what we did in metadata, i create a new class named "TsFileSyncManager", all deletion, new TsFile will be sent to the TsFileSyncManager derectly, and TsFileSyncManager will transfer the data to TsFilePipe.
2. the collect process of history TsFiles is moved to TsFileManager in VSGP, for not be interfered by compaction.
3. to avoid collect duplicate TsFile, i redesign the process of collect history TsFile, for more details, please check TsFileSyncManager#registerAndCollectHistoryTsFile.

Reply:
MetadataSyncManager has been rename to SchemaSyncManager.